xmldsig/README.md

```diff
@@ -0,0 +1,61 @@
+# XMLDSIG verification in every programming language
+
+This repository serves to show how to verify XMLDSig signature in every programming language by producing a dsigverifier commandline utility in as any programming languages as possible. It's use is not limited to TokenScript, but it may serve a starting point for anyone who wish to implement TokenScript, as the first thing you do with them is, typically, to verify its signature.
+
+## Usage
+
+In all following examples, assume you have cloned [TokenScript-Repo](https://github.com/AlphaWallet/TokenScript-Repo) - an incomplete TokenScript collection - into your home directory (`~/TokenScript-Repo`) for testing purpose.
+
+
+### JavaScript:
+
+First, install the dependencies:
+
+    $ cd js
+    $ npm install
+
+Then run node.js:
+
+    $ node xmldsigverifier.js ~/TokenScript-Repo/aw.app/2020/06/*
+
+Note that at this stage half of the tokenscripts fails validation due to a bug in a dependent JavaScript libraries.
+
+Sample output:
+
+    [FAILED] TokenScript-Repo/aw.app/2020/06/COMP.tsml
```

Comment:
@colourful-land If I am to see that this does not work properly would you like to make it super clear that this is the case? e.g. this is a sample JS library but it doesn't work properly and is only intended as an example.

Reply:
highlighted the relevant part (that says it doesn't work properly due to a bug in a dependent library that is 𝑛𝑜𝑡 in this repo).